src/test/java/org/openmrs/contrib/qaframework/automation/EncountersSteps.java

```diff
@@ -0,0 +1,101 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.AdmitToInpatientPage;
+import org.openmrs.reference.page.PatientVisitsDashboardPage;
+import org.openmrs.uitestframework.test.TestData;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class EncountersSteps extends Steps {
+
+	private TestData.PatientInfo testPatient;
+
+	@Before(RunTest.HOOK.SELENIUM_ENCOUNTER)
+	public void visitHomePage() {
+		testPatient = createTestPatient();
+		initiateWithLogin();
+	}
+
+	@After(RunTest.HOOK.SELENIUM_ENCOUNTER)
+	public void destroy() {
+		deletePatient(testPatient);
+		quit();
+	}
+
+	@Given("a user clicks on Find Patient app from Home page")
+	public void goToFindPatientRecordPage() {
+		findPatientPage = homePage.goToFindPatientRecord();
+	}
+
+	@Then("the system loads find patient record page")
+	public void systemloadsFindPatientRecordPage() {
+		assertTrue(textExists("Find Patient Record"));
+	}
+
+	@When("a user searches an existing patient and starts the visit")
+	public void searchExistingPatient() {
+		findPatientPage.enterPatient(testPatient.identifier);
+		findPatientPage.waitForPageToLoad();
+		visitsDashboardPage = findPatientPage.clickOnFirstPatient()
+				.startVisit();
+	}
+
+	@Then("the system loads patient visit dashboard page")
+	public void systemloadsVisitDashboardPage() {
+		assertTrue(visitsDashboardPage.containsText("Visits"));
+	}
+
+	// Admission encounter
+	@When("a user clicks on Admit to Inpatient button and selects location")
+	public void goToAdmitToInpatientPage() {
+		visitsDashboardPage.goToAdmitToInpatient().confirm("Inpatient Ward");
+	}
+
+	@And("the system confirms admission encounter is made")
+	public void confirmAdmissionEncounter() {
+		assertTrue(visitsDashboardPage.containsText("Admission"));
+		assertThat(visitsDashboardPage.getEncountersCount(), is(1));
+	}
+
+	// Transfer encounter
+	@When("a user clicks on Trasfer to Ward service button and selects location")
+	public void goToTrasferToWardServicePage() {
+		visitsDashboardPage.goToTransferToWardServicePage().confirm(
+				"Isolation Ward");
```

Comment:
Can this be in one line as well.

Reply:
Refer to the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/81#discussion_r693478959)